app/scripts/services/StlCourtsDatepickerInputValidation.svc.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+angular.module('yourStlCourts').factory('StlCourtsDatePickerInputValidation', function () {
```

Comment:
What's this service for? It doesn't appear to be used anywhere and I question why it would be if WERE used somewhere